gensim/models/phrases.py

```diff
@@ -133,19 +133,24 @@ def __init__(self, sentences=None, min_count=5, threshold=10.0,
         `delimiter` is the glue character used to join collocation tokens, and
         should be a byte string (e.g. b'_').
 
+        `recode_to_utf8`- By default, the input sentences will be internally encoded to
+        UTF-8 bytestrings, to save memory and ensure valid UTF-8. Set recode_to_utf8=False
```

Comment:
Best to render code blocks as literal text (put `recode_to_utf8=False` in backticks).